bodo/pandas/_physical_conv.cpp

```diff
@@ -391,3 +391,17 @@ void PhysicalPlanBuilder::Visit(duckdb::LogicalCopyToFile& op) {
     finished_pipelines.emplace_back(this->active_pipeline->Build(physical_op));
     this->active_pipeline = nullptr;
 }
+
+void PhysicalPlanBuilder::Visit(duckdb::LogicalDistinct& op) {
```

Comment:
Make sure `op.order_by` is not set? Example DuckDB code: https://github.com/duckdb/duckdb/blob/d29a92f371179170688b4df394478f389bf7d1a6/src/optimizer/pushdown/pushdown_distinct.cpp#L10

Reply:
Good catch.  I originally wrote this code before we decided to be careful about these unchecked fields.